README.md

```diff
@@ -212,6 +212,12 @@ we suggest you check out the following:
 + Interactive Workshop: https://github.com/jlord/git-it-electron
 + [@NataliaLKB](https://github.com/NataliaLKB)'s Tutorial: https://github.com/NataliaLKB/learn-git-basics
 
+### Design
+
+Building something for _someone else_ is hard. You don't necessarily know that people. Worrying about someone else's _"experience"_ with your app can be a full-time job. However, it is an essential process for everyone involved in software to partake in. The very best pieces of software are differentiated by the quality of their **User Experience design**.
```

Comment:
As @newswim mentioned, "that people" :arrow_right: "that person" or "those people"